<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Сапёр</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lvl1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>1 Уровень</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lvl2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>2 уровень</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Сапёр тайм-аут</string>
   </property>
  </widget>
  <widget class="QPushButton" name="time_lvl1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>1 уровень</string>
   </property>
  </widget>
  <widget class="QPushButton" name="time_lvl2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>2 уровень</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lvl3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string>3 уровень</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
